client/aws/errors.go

```diff
@@ -11,8 +11,23 @@ const (
 	ELBConfigurationMismatch = "already exists and it is configured with different parameters"
 )
 
-var (
-	malformedAmazonAccountIDError   = errgo.New("amazon account ID can only contain numbers")
-	wrongAmazonAccountIDLengthError = errgo.New("amazon account ID has the wrong size")
-	emptyAmazonAccountIDError       = errgo.New("amazon account ID cannot be empty")
-)
+var malformedAmazonAccountIDError = errgo.New("amazon account ID can only contain numbers")
```

Comment:
Haha, this will be long 🍿 
I ❤️ @xh3b4sd efforts for that to be standardised though.
Wiki maybe?